handbook/engineering/README.md

```diff
@@ -36,6 +36,20 @@ To [create an engineering-initiated user story](https://fleetdm.com/handbook/eng
 
 > We prefer the term engineering-initiated stories over technical debt because the user story format helps keep us focused on our users and contributors.
 
+### Manage release branches
+Every three weeks, release a minor version of Fleet from the `main` branch. 
+
+Every week between minor releases, release a patch release with fixes for released bugs. 
+
+The DRI for publishing a release is responsible for creating the next patch branch. Create a patch branch off the latest tagged release of Fleet. Merge released bug fixes directly into the target patch branch to avoid merge conflicts later in the release cycle. After merging into the patch branch, submit a PR to `main` containing the same fix and resolve any merge conflicts.
```

Comment:
my only concern is about logistics, in these kinds of flows generally you get a PR up first and then "backport" the change to the patch branch.

I don't know how strict we want to be, but as it's worded right now:

> After merging into the patch branch, submit a PR to `main` containing the same fix and resolve any merge conflicts.

By the time you get code review comments in your PR, the commit is already merged into the patch. Unless you make two PRs, which might introduce too much friction.

Reply:
Looking at the other review comments seems like the implicit intention is to make two PRs. I think it might be worth to make that explicit here.

Sharing what I meant by too much friction in case it helps:

1. CI generally takes 40 minutes to run for back-end changes
2. Fixing merge conflicts requires another review for each codeowner + another CI run

Reply:
@roperzh Thanks for your feedback! You're right; the language did imply that commits should be made directly to the patch branch, but I meant two PRs should be opened. I've revised the language to make that more clear. 

Unfortunately, this will introduce more friction in the form of two PRs. The benefit is that we won't have to deal with merge conflicts when we're cherry-picking commits into the patch branch, which sometimes becomes a big distraction for the team. 

We also found ourselves in a situation with 4.46.3 where we couldn't release a fix without rewriting it because it had been built on top of feature code that was not included in the patch. That was the motivation to make this change. 

40 minutes for CI is a long time. I'll look into that, and see if we can prioritize some efforts to optimize it.

Reply:
@lukeheath that makes sense, thank you!

Reply:
I agree with @roperzh 's concern of the 2 PRs overhead. One thing that could make the process smoother is if we could avoid requiring approval again for the 2nd PR, given that it was already reviewed/approved when targeting the patch branch. Creating a PR knowing that it can be merged immediately once CI is done is not too big of an overhead vs having it reviewed again, possibly by different persons and new/different comments to address, with the additional CI runs that this implies. Don't know how feasible that is (ideally it detects that it's the same commit already reviewed/merged in the patch to bypass the approval requirements), but it would be nice.

But in any case we can live with it a bit first and see how much of a pain it is.

Reply:
Agreed. The only catch would be if there are merge conflicts going in `main`. 

@sharon-fdm is creating an engineering-initiated story for a GitHub action that would automatically open a PR back to `main` for any patch branch commits. Maybe possible for that action to auto-approve?

Reply:
@lukeheath [17603](https://github.com/fleetdm/fleet/issues/17603)